### ISSUE TITLE
wifi restart

### ISSUE THREAD

**Issue by Kristian8606** [maximkulkin/esp-wifi-config#23]
When I restart my router, I notice that my thermostats are restarting.
I still have not checked what causes the restart and first decided to ask if it is possible for the library to restart the device when I restart the wifi router?

**Comment by Masterpc96** [comment]
Hi I worked on the other project using another WiFi library and as I know it is normal because esp has no connection to stored WiFi so library restart esp to try to connect to WiFi. If still there is no connection it creates access point

**Comment by maximkulkin** [comment]
No, this is actually not an intended behavior as library will go into AP mode if it can't connect to configured network and it will continue trying to connect.
